### ISSUE TITLE
TS Syntax error in app/types/app-extension.d.ts line (14,15) 

### ISSUE THREAD

**Issue by dinesh-chander** [quasarframework/quasar#12080]
### What happened?

On running quasar dev, TS is throwing syntax errors for lines 14 and 15 in 
app/types/app-extension.d.ts

https://github.com/quasarframework/quasar/blob/05f407a70ee9c6d916a4e557562d3ff6e0c3bbf1/app/types/app-extension.d.ts#L14

### What did you expect to happen?

Not required

### Reproduction URL

Not required

### How to reproduce?

use @quasar/app package version 3.2.7 and run quasar run dev -m ssr

### Flavour

Quasar CLI (@quasar/cli | @quasar/app)

### Areas

Quasar CLI Commands/Configuration (@quasar/cli | @quasar/app)

### Relevant log output

```shell
App •  ERROR  •  Webserver  in app\node_modules\@quasar\app\types\app-extension.d.ts

[tsl] ERROR in app\node_modules\@quasar\app\types\app-extension.d.ts(14,121)
      TS1005: ';' expected.

 App •  ERROR  •  Webserver  in app\node_modules\@quasar\app\types\app-extension.d.ts

[tsl] ERROR in app\node_modules\@quasar\app\types\app-extension.d.ts(15,123)
      TS1005: ';' expected.
```

**Comment by yusufkandemir** [comment]
Result of #9758.

**Comment by yusufkandemir** [comment]
Fixed in https://github.com/quasarframework/quasar/commit/2ba8fcfd6ae09a17d95ae400e78d15413d7d4ca9

**Comment by IlCallo** [comment]
Definitely my fault, haven't catched those commas from GH GUI after last review iteration
Seems like GH automerge also messed up with q/app devDeps and had to re-add those too 😅 

Will be fixed into upcoming @quasar/app v3.2.8
Thanks for reporting :)
